### PR TITLE
feat: make folder structure based on FSD

### PR DESCRIPTION
**Pull request by marrollyn**

### CODE REVIEW

**Review by nookismile — COMMENTED** [review]
В styles должны быть файлы .scss, сейчас - index.html

File: src/02_pages/addBookPage/index.html
Comment (MariiaBel) [inline src/02_pages/addBookPage/index.html:1]: - Could you rename manePage to homepage? Usually it  names homepage. 
- We do not have addbookpage and bookpage. 
- Could you add catalog page to this folder?

File: src/03_widgets/adminLoginModal/index.html
Comment (MariiaBel) [inline src/03_widgets/adminLoginModal/index.html:1]: I think we will have many modals. That why I suggest you create modal folder with adminLogin inside the modal folder

File: src/05_entities/book/index.html
Comment (MariiaBel) [inline src/05_entities/book/index.html:1]: Rename this folder to `document` . It will be pdf or image document

File: src/05_entities/filter/index.html
Comment (MariiaBel) [inline src/05_entities/filter/index.html:1]: filters is for actions with documents/ I think it needs to be in feature folder

**Review by MariiaBel — APPROVED** [review]

**Pull request merged**